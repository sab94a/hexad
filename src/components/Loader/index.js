// @flow

import React from 'react';
import CircularProgress from '@material-ui/core/CircularProgress';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  progress: {
    margin: theme.spacing(2),
  },
}));

const Loader = () => {
    const classes = useStyles();

    return (
        <CircularProgress className={classes.progress} />
    );
}

export default Loader;
