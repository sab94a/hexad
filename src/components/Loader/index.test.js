import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Loader from './'

describe('Loader', () => {
    it('Should render correctly', () => {
        const component = shallow(<Loader />);
        expect(shallowToJson(component)).toMatchSnapshot();
    });
})
