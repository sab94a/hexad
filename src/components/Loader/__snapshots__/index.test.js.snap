// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Should render correctly 1`] = `
<div>
  Loading...
</div>
`;
